### DISCUSSION TITLE
How to disable minification of JavaScript and CSS?

### DISCUSSION THREAD

**Discussion by n0lr4m** [squidfunk/mkdocs-material#2950] [Q&A] [answered]
Hello, 

Test build a site compatible with SharePoint, that been said there is a way to disable minify for CSS and Javascript?

Thanks,

Marlon

**Comment by squidfunk** [comment, marked as answer]
Yes, you can compile the theme yourself without the `--optimize` flag, which will disable minification:

https://github.com/squidfunk/mkdocs-material/blob/13f6ce4a5bfe62313790b1ab2e26ca20401cf77e/package.json#L27

**Reply by n0lr4m** [reply]
Thank you!

I was able to compile the theme following these steps: https://squidfunk.github.io/mkdocs-material/customization/#theme-development


Cheers!

**Comment by n0lr4m** [comment]
Ok, it seems that minifying isn't the issue with SharePoint but browse compatibility IE and Edge more specifically.

So create a main.html on a custom_html folder to "inject" the meta tag for browser compatibility into the header. This is the best way to do it?

![image](https://user-images.githubusercontent.com/63619741/130789431-329838d2-92d3-4593-899a-7d7210b21ed5.png)

**mkdocs.yaml:**

```
site_name: MKDocs SharePoint
theme:
  name: material
  custom_dir: custom_html
site_url: ""
use_directory_urls: false
```
**main.html:**

```
% extends "base.html" %}
{% block site_meta %}
<meta http-equiv="X-UA-Compatible" content="IE=edge">
{% endblock %}
```

**Reply by squidfunk** [reply]
Yes. Note that Internet Explorer support was dropped in version 5.

Here are some directions on what would need to be done in order to make it work again:

- #1537
- #1464#issuecomment-594409955

**Reply by n0lr4m** [reply]
I see.

Thanks,

Marlon
